installer_scripts/install-yb-voyager

```diff
@@ -645,15 +645,15 @@ output() {
 install_golang() {
 	if [ -x ${GO} ]; then
 		gov=`go version | { read _ _ v _; echo ${v#go}; }`
-		version_ok=$(version_satisfied 1.20 "$gov")
+		version_ok=$(version_satisfied 1.23.0 "$gov")
 		if [[ $version_ok -eq 1 ]]
 		then
 			output "Found golang=${gov}"
 			return
 		fi
 	fi
 	export GOROOT="/tmp/go"
-	GO_VERSION="go1.20"
+	GO_VERSION="go1.23.0"
```

Comment:
@ShivanshGahlot, I think you should also include the changes in the code base required for upgrading in the same PR.

Reply:
I have included those changes.